### ISSUE TITLE
kubernetes.el fails to build

### ISSUE THREAD

**Issue by NeQuissimus** [NixOS/nixpkgs#62834]
## Issue description

The emacs package "kubernetes" fails to build.

```
Done (Total of 27 files compiled, 2 skipped)
Searching for program: No such file or directory, git
builder for '/nix/store/walfx3q38f4vr9l94i0fyj4m3c71swnl-emacs-kubernetes-20190327.1744.drv' failed with exit code 255
error: build of '/nix/store/walfx3q38f4vr9l94i0fyj4m3c71swnl-emacs-kubernetes-20190327.1744.drv' failed
```

### Steps to reproduce

`nix-build -A emacs26PackagesNg.kubernetes`

## Technical details

```
λ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.1.7-hardened, NixOS, 19.09.git.7dc8413 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2.2`
 - channels(root): `"nixos-19.09pre182062.1dc26c32edf"`
 - channels(nequi): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by masaeedu** [comment]
Seeing this while trying to build a totally unrelated package (`ghc`) as well. In my case the workaround was to add `git` as a `buildInput` explicitly using `overrideAttrs`, but I don't have any idea how this used to work before.

**Comment by adisbladis** [comment]
Fixed in f63dbf3c94fcf53231996c99babaf509a365df00
